src/tests/config/global_configs.py

```diff
@@ -17,22 +17,23 @@
 global_variables = DefaultVariables()
 
 
-@dataclass
-class ClusterConfig(BaseClusterConfig):
-    """A Cluster configuration with defaults that obtained from EnvConfig"""
-
-    @staticmethod
-    def get_default(key, default=None) -> Any:
-        return getattr(global_variables, key)
-
+class _EntityISOPath(ABC):
     @staticmethod
-    def _get_iso_download_path(cluster_name: str):
+    def _get_iso_download_path(entity_name: str):
         return str(
-            Path(env_defaults.DEFAULT_IMAGE_FOLDER).joinpath(f"{cluster_name}-{env_defaults.DEFAULT_IMAGE_FILENAME}")
+            Path(env_defaults.DEFAULT_IMAGE_FOLDER).joinpath(f"{entity_name}-{env_defaults.DEFAULT_IMAGE_FILENAME}")
         ).strip()
 
+
+@dataclass
+class ClusterConfig(BaseClusterConfig, _EntityISOPath):
```

Comment:
we're having multiple inheritance just for using a utility function?
I think ``_get_iso_download_path`` can become something a lot simpler, not sure this complexity is justified